CliClient/locales/zh_CN.po

```diff
@@ -627,16 +630,15 @@ msgid "Cancelling... Please wait."
 msgstr "正在取消... 请稍候。"
 
 #: /Users/tessus/data/work/joplin/Tools/../CliClient/app/command-tag.js:14
-#, fuzzy
 msgid ""
 "<tag-command> can be \"add\", \"remove\", \"list\", or \"notetags\" to "
 "assign or remove [tag] from [note], to list notes associated with [tag], or "
 "to list tags associated with [note]. The command `tag list` can be used to "
 "list all the tags (use -l for long option)."
 msgstr ""
-"<tag-command> 可以是“add”、“remove”或者“list”， 用于从 [note] 中赋值或删除 "
-"[tag]，或者列出与 [tag] 相关的笔记。`tag list` 命令可以用于列出所有的标签（对"
-"于过长选项请使用 -l 参数）。"
+"<tag-command> 可以是“add”、“remove”、“list” 或者 “notetags”， 用于从 [note] "
+"中赋值或删除 [tag]，或者列出与 [tag] 相关的笔记。`tag list` 命令可以用于列出"
+"所有的标签（对于过长选项请使用 -l 参数）。"
```

Comment:
Somehow the use of commas is a bit inconsistent. Do you want to use `、` or `，`? Please pick one.

As I could see from the rest of the translation, it looks like that `，` or `,` was used.

Reply:
Thank you for your reminder! But in fact the usage here is correct. In Chinese, when enumerating three or more things, we need to use "、". Also, I have checked other related punctuation issues and made some minor changes.

Reply:
I was just puzzled that there were 2 different commas. But if this is ok in Chinese, all is good.